src/_common/lightbox/item/item.styl

```diff
@@ -48,12 +49,21 @@ img
 	width: 100%
 	position: relative
 
+	// Make room for slider controls on Xs breakpoint - helps with issues like iframes
+	// breaking vue-touch and allows us to use arrows as an indication of more content.
+	@media $media-xs
```

Comment:
I kind of think we should show the arrows at the bottom of the lightbox window on mobile if we are gonna show them, instead of left/right of the media item. The wider we can show an image, the better, I think. What are your thoughts on that?